doc/src/build/fullnode.md

```diff
@@ -380,6 +380,19 @@ state-archive-read-config:
     # Whether to prune local state based on latest checkpoint in archive.
     # This should stay false for most use cases
     use-for-pruning-watermark: false
+  - object-store-config:
```

Comment:
Could we publish it as two different configs? Don't want to confuse someone who'd copy-paste the whole text as a single config. Just say that this is an example for a fallback to AWS archive, this is an example for a fallback to GCS archive, or something like that.

Reply:
sure, split it into two different sections